docs/Technical-Documentation/buildpack-changelog.md

```diff
@@ -1,4 +1,6 @@
 ## Buildpacks Changelog
+- 05/15/2023 [nginx-buildpack v1.2.2](https://github.com/cloudfoundry/nginx-buildpack.git#v1.2.2)
```

Comment:
Nit: I always like to go with one version lower than current, but if you want there is a newer buildpack version available as of May 15th ->v1.2.3